store/mockstore/mockcopr/cop_handler_dag.go

```diff
@@ -921,12 +922,18 @@ func extractOffsetsInExpr(expr *tipb.Expr, columns []*tipb.ColumnInfo, collector
 
 // fieldTypeFromPBColumn creates a types.FieldType from tipb.ColumnInfo.
 func fieldTypeFromPBColumn(col *tipb.ColumnInfo) *types.FieldType {
+	charsetStr, collationStr, err := charset.GetCharsetInfoByID(int(collate.RestoreCollationIDIfNeeded(col.GetCollation())))
+	if err != nil {
```

Comment:
Need to log an error level message.

Reply:
The behavior is same as before :(

`collate.CollationID2Name` also didn't have error log

Reply:
Then please fix it here.

Reply:
https://github.com/pingcap/tidb/blob/2a89d126bcdf1de20cc89e7d3c8e40c039b8914a/util/collate/collate.go#L177-L184
It seems comment by @bb7133 , because of `repeating logs`. Not sure we should add it or not.